test/ocamlorg_web/graphql_test.ml

```diff
@@ -145,48 +139,60 @@ let state_test () =
 let () =
   Alcotest.run
     "ocamlorg"
-    [ ( "test that starts_with function compares two similar strings"
+    [ ( "test that get_info function returns all the dependencies"
       , [ Alcotest.test_case
-            "and returns true"
+            "and returns all 6 dependencies"
             `Quick
-            (start_with_test "OC" "oc" true)
+            get_info_test
         ] )
-    ; ( "test that starts_with function compares two different strings"
+    ; ( "test that is_in_range function checks if a current version is within \
+         the start and last version range specified"
       , [ Alcotest.test_case
-            "and returns false"
+            "and returns true"
             `Quick
-            (start_with_test "gv" "Bt" false)
+            (is_in_range_test "3.12.0" "2.0.0" "5.9.2" true)
```

Comment:
Since you're using `is_in_range` on type `Package.Version.t`, it might be better to also test it on that type instead of testing it on `string`. What do you think?

Reply:
I am noticing a mix up here, the new test is not supposed to have `starts_with_test` as it has been removed. I will update in the next push

Reply:
I think that's ok: `starts_with_test` is in red, so that means it was removed. What I was referring to in the comment was the line 
```
(is_in_range_test "3.12.0" "2.0.0" "5.9.2" true)
```
In that line, you're testing the function `is_in_range` on the strings `"3.12.0"`, `"2.0.0"` and `"5.9.2"`. You [use it in the code](https://github.com/ocaml/v3.ocaml.org-server/pull/78/files#diff-7b9c15d336b381008174d3a481ce2b8f7812523a2ee852caf532f4fa332857feR117-R120) on type `Paclage.Version.t` though (which is the right thing to do). So I was suggesting to also test it on type `Paclage.Version.t` instead of testing it on type string.

Reply:
Done